src/app/collection/pages/collection-ncyte/stats/stats.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { CollectionService } from 'app/core/collection.service';
+
+
+@Component({
+  selector: 'clark-stats',
+  templateUrl: './stats.component.html',
+  styleUrls: ['./stats.component.scss']
+})
+export class StatsComponent implements OnInit {
+
+  constructor(private collectionService: CollectionService) {
+
+   }
+
+  public collectionMetrics;
+  public authorCollection;
+  public objDownload;
+  public objReview;
+  public objReleased;
+
+  ngOnInit(): void {
+    this.collectionService.getCollectionMetricsData('ncyte').then((res: any) => {
+      this.collectionMetrics = res;
```

Comment:
If this variable isn't being used in the html ( {{ this syntax }} ) then this variable can be deleted.